From bb43daa4d4df4200dfa07b7be52efcf2ec536be9 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 16 Oct 2023 12:16:46 +0200
Subject: [PATCH 0005/1695] core: arm: add notice to sm/psci.h and sm/std_smc.h

Adds missing notice in sm/psci.h and sm/std_smc.h that was not stated
when these header files were initially merged [1]. License is set
to BSD-2-Clause as per OP-TEE OS license model at the time of [1] and
up to now. Copyright is set to Linaro as per being the maintainer of
OP-TEE.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 5c7c2018231ae57dec27932b594a8ce29cfc8ffe core: arm: add notice to sm/psci.h and sm/std_smc.h]
Change-Id: Ic5ee402bc64f4c25bc2aca5ac80563abf847d746
Link: https://github.com/OP-TEE/optee_os/commit/319556cdc38514d0261bec0abf87627d0782f8e [1]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 2441aaf021490133e9f84bb57bc39db7f708308f]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/include/sm/psci.h    | 4 ++++
 core/arch/arm/include/sm/std_smc.h | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/core/arch/arm/include/sm/psci.h b/core/arch/arm/include/sm/psci.h
index b992e60ae..da873615c 100644
--- a/core/arch/arm/include/sm/psci.h
+++ b/core/arch/arm/include/sm/psci.h
@@ -1,3 +1,7 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2017-2023, Linaro Limited
+ */
 #include <kernel/thread.h>
 #include <sm/sm.h>
 #include <stdint.h>
diff --git a/core/arch/arm/include/sm/std_smc.h b/core/arch/arm/include/sm/std_smc.h
index ce8606dcf..8c55ec72c 100644
--- a/core/arch/arm/include/sm/std_smc.h
+++ b/core/arch/arm/include/sm/std_smc.h
@@ -1,3 +1,7 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2017-2023, Linaro Limited
+ */
 #ifndef __STD_SMC_H__
 #define __STD_SMC_H__
 
-- 
2.47.2

